assets/src/scripts/components/hydrograph/layout.js

```diff
@@ -38,12 +41,19 @@ const SPARK_LINE_DIM = {
 const layoutSelector = createSelector(
     (state) => state.width,
     (state) => state.windowWidth,
-    (width, windowWidth) => {
+    tickSelector,
+    (width, windowWidth, tickDetails) => {
+        const margin = mediaQuery(USWDS_SITE_MAX_WIDTH) ? MARGIN : MARGIN_SMALL_DEVICE;
+        const tickLengths = tickDetails.tickValues.map(v => tickDetails.tickFormat(v).length);
+        const approxLabelLength = Math.max.apply(null, tickLengths) * 10;
```

Comment:
I would have opted for `Math.max(...arr)` here, but tomayto, tomahto. :man_shrugging:

Reply:
I had been erroring on the side of "performance", because I had read somewhere that the spread operator is expensive... but apparently that's only with Traceur, not Babel: http://incaseofstairs.com/2015/06/es6-feature-performance/ Might as well use the spread operator everywhere.